services/service-core/src/app.module.ts

```diff
@@ -53,9 +54,9 @@ import { KeycloakAuthGuard } from './auth/keycloak.guard';
     KeycloakConnectModule.registerAsync({
       imports: [ConfigModule],
       useFactory: async (configService: ConfigService) => ({
-        authServerUrl: configService.get('KEYCLOAK_SERVER_URL'),
-        realm: 'humanitech',
-        resource: 'nest-application',
+        authServerUrl: configService.get('keycloakServerUrl'),
+        realm: configService.get('keycloak.realm'),
+        resource: configService.get('keycloak.nestClientId'),
         secret: configService.get('KEYCLOAK_SECRET'),
         'public-client': true,
         verifyTokenAudience: true,
```

Comment:
This can be more by clean by using object destructuring like
```typescript
       const {realm,  nestClientId, KEYCLOAK_SECRET  } = configService.get('keycloak');
``` 
and you can pass the values directly